apps/mobile/src/components/ChatDetailScreen/components/OtherSideNamePhotoAndInfo.tsx

```diff
@@ -18,12 +18,14 @@ function OtherSideNamePhotoAndInfo({mode}: Props): JSX.Element {
     otherSideDataAtom,
     otherSideLeftAtom,
     canSendMessagesAtom,
+    commonConnectionsHashesAtom,
   } = useMolecule(chatMolecule)
 
   const offer = useAtomValue(offerForChatAtom)
   const otherSideData = useAtomValue(otherSideDataAtom)
   const otherSideLeft = useAtomValue(otherSideLeftAtom)
   const canSendMessages = useAtomValue(canSendMessagesAtom)
+  const commonConnectionsHashes = useAtomValue(commonConnectionsHashesAtom)
```

Comment:
Could you make an atom specificaly for length? Ideally selectAtom to prevent rerenders